Dockerfile.build

```diff
@@ -1,11 +1,17 @@
 FROM quay.io/centos/centos:stream8
 
-ENV VIRTUAL_ENV=/opt/venv
-ENV PATH="$VIRTUAL_ENV/bin:$PATH"
-RUN dnf update -y && dnf install -y epel-release &&\
+ENV VIRTUAL_ENV=/opt/venv\
+    PATH="$VIRTUAL_ENV/bin:$PATH"
+
+RUN --mount=type=tmpfs,destination=/var/cache\
+    --mount=type=tmpfs,destination=/root/.cache\
+    --mount=type=cache,target=/var/cache/yum\
+    --mount=type=cache,target=/var/cache/dnf\
+    --mount=type=cache,target=/root/.cache\
```

Comment:
what is a advantage at having same directories tmpfs and cache?

Reply:
From my tests I could observe that cache saves it as cache in some storage, so it can take advantage for future usage
tmpfs makes it ephemeral, so no need to `dnf clean all` (which would delete them from the cache thus making cache useless)

Reply:
ok so they are not exclusive, it not like having 2 mount points for the same directory,